vxvas2nets/vas2nets_sms.py

```diff
@@ -213,12 +208,13 @@ def handle_outbound_message(self, message):
             yield self.handle_send_timeout(message)
             return
 
-        # NOTE: we are assuming here that they send us a non-200 response for
-        # error cases (this is not mentioned in the docs)
-        if resp.code == http.OK:
-            returnValue((yield self.handle_outbound_success(message, resp)))
```

Comment:
I think we should still return an error if the response code is not 200 (e.g. we might get a gateway timeout or other error that is not from their server itself), and then also check the content does not look like an error.